WebContent/WEB-INF/jsp/stats.jsp

```diff
@@ -101,8 +101,8 @@
 			<li><div class="item"><a href="statsKeywords.do">DAMS Keywords/Phrases</a></div></li>
 			<c:if test="${model.isCas}"><li><div class="item"><a href="collectionsReport.do">DAMS Collections</a></div></li></c:if>
 			<c:if test="${model.isCurator}">
-				<li><div class="item"><a href="statsRdcpUsage.do">RDCP Objects Views by Month</a></div></li>
-				<li><div class="item"><a href="statsRdcpDownload.do">RDCP File Download By Month</a></div></li>
+				<li><div class="item"><a href="statsRdcpUsage.do?unique">RDCP Objects Views by Month</a></div></li>
```

Comment:
I'm curious why this `unique` property is needed? Is `statsRdcpUsage.do` called somewhere else where it does not require the new unique views support?

Reply:
@mcritchlow I am adding the support that we can switch to and from the unique views and pages views with unique views the default. And the `unique` property is just serving  as a flag for this purpose.

Reply:
I see, ok thanks for clarifying. I just wasn't sure on whether non-unique views were something they still wanted. if they are, then nevermind :)